deltachat-ios/Chat/ChatViewController.swift

```diff
@@ -1042,17 +1042,17 @@ class ChatViewController: UITableViewController {
 
         let rectOfCellInTableView = tableView.rectForRow(at: lastIndexPath)
         // convert points to same coordination system
-        let inputBarTopInWindow = messageInputBar.convert(CGPoint(x: 0, y: messageInputBar.bounds.minY), to: window)
+        let inputBarTopInWindow = window.bounds.maxY - (messageInputBar.intrinsicContentSize.height + messageInputBar.keyboardHeight)
```

Comment:
is it really correct to remove the convert() function here?

further down, we are comparing a convert()'ed value with an unconverted.

just wondering, i am not saying it is wrong :)

Reply:
it should be fine, all views are converted to the coordination system of the window. the new change also calculates the coordinates based on the window coordination system, however in a different way.

Reply:
k, then i assume, it is correct :)

thanks for explanations!